items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -0,0 +1,64 @@
+package weco.api.items.services
+
+import grizzled.slf4j.Logging
+import weco.api.stacks.models.SierraItemIdentifier
+import weco.api.stacks.services.SierraService
+import weco.catalogue.internal_model.identifiers.{IdState, IdentifierType}
+import weco.catalogue.internal_model.locations.{
+  AccessCondition,
+  PhysicalLocation
+}
+import weco.catalogue.internal_model.work.Item
+import weco.catalogue.source_model.sierra.identifiers.SierraItemNumber
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SierraItemUpdater(sierraService: SierraService)(
+  implicit executionContext: ExecutionContext
+) extends ItemUpdater
+    with Logging {
+
+  val identifierType = IdentifierType.SierraSystemNumber
+
+  private def updateLocations(
```

Comment:
Minor: I'd call this updateAccessCondition, maybe, which describes what it does – updateLocations is more of an implementation detail.